src/main/kotlin/no/nav/helse/dokument/Dokument.kt

```diff
@@ -22,6 +24,9 @@ data class Dokument(
         var result = title.hashCode()
         result = 31 * result + content.contentHashCode()
         result = 31 * result + contentType.hashCode()
+        result = 31 * result + eier.hashCode()
         return result
     }
-}
\ No newline at end of file
+}
+
+data class DokumentEier(val eiersFødselsnummer: String)
```

Comment:
kalle verdien her `identitetsnummer` og det Fødselsnummer-aliaset for Identitetsnummer.
Også blir det litt smør på flesk å gjenta objektets navn foran variabelen her

Reply:
Den er blitt sånn pga følgende:
POST /v1/dokument/ey...
```json
{
    "eiers_fødselsnummer": "12345678910"
}
```
Følte at det var litt mer forklarende :p 
Endringen av verdien i DokumentEier medfører også endringer i k9-mellomlagring, men det burde ikke være noe stor sak å få til. Skal jeg skrive om?